post_processing/msd.py

```diff
@@ -264,12 +262,13 @@ def save_MSD_time_ave(simpath, D, savepath, every_other=10):
     every_other : int
         skip every_other conformation when loading conformations for MSD computation
     """
-    Xhot, Xcold = extract_hot_cold(
-        Path(simpath), D, start=100000, every_other=every_other
-    )
-    hot_msd, cold_msd = get_bead_msd_time_ave(Xhot, Xcold)
-    df_msd = pd.DataFrame()
-    df_msd["Times"] = np.arange(0, len(hot_msd)) * every_other
-    df_msd["MSD_A"] = hot_msd
-    df_msd["MSD_B"] = cold_msd
-    df_msd.to_csv(savepath)
+    if not Path(savepath).exists():
```

Comment:
not sure what you're trying to do here. Is savepath the parent directory of simpath? If so, you can just do "if not Path(simpath.parent).exists()"